spec/support/cli.cr

```diff
@@ -0,0 +1,107 @@
+Spec.before_each do
+  application_path.tap do |path|
```

Comment:
Why tap?

Reply:
Didn't notice that in the code migration. I'll remove it.